### DISCUSSION TITLE
Please add video title & link option to CLI Thanks

### DISCUSSION THREAD

**Discussion by BillionShields** [jdepoix/youtube-transcript-api#216] [Ideas]
Would be awesome if in 1st 2 lines of  transcript you can include title and full URL to the video 
Example
================
The Growing Megafire Crisis – and How to Contain It | George T. Whitesides | TED
https://www.youtube.com/watch?v=M66jkJMH3DI

My wife grew up in Santa Rosa, California, and her best friend was a woman named Joy Durand who lived on the northeast outskirts of town. At midnight on the evening of October 8, 2017, Joy's telephone ran.....

**Comment by jdepoix** [comment]
This is more of a feature request than a discussion and you already created an issue for it. Therefore, I will close this discussion.
